cps/kni.c

```diff
@@ -993,13 +993,20 @@ init_kni(const char *kni_kmod_path, unsigned int num_kni)
 
 	ret = init_module(file, len, "");
 	if (ret < 0) {
+		if (errno == EEXIST) {
+			CPS_LOG(NOTICE, "%s: %s already inserted\n",
+				__func__, kni_kmod_path);
```

Comment:
Could you find the version of the already loaded kernel module and add this information to this log entry. See the discussion on #96 to understand why we need this.

You can take a look at `modinfo` to find how to get this information. The following three fields should be logged:
```
version:        3.102
srcversion:     29B86B4EFF2E078A71F716A
vermagic:       2.6.32-5-amd64 SMP mod_unload modversions
```
I got the example above on the following page:
https://www.cyberciti.biz/faq/linux-check-version-of-ko-kernel-module-driver/

Adding a thrid patch to this pull request in order to add the version information would be nice, so this patch would be ready for merge. But the new patch would close #96 instead of this.